.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -43,3 +43,29 @@ Relevant software versions:
 ## Additional context
 
 <!-- Add any other context about the problem here, such as stack traces or debugging info. -->
+
+#### Definition of Done
+
+- [ ] This bug is **done**:
+  - [ ] Bug resolved to **user's** satisfaction
+  - [ ] Automated tests are passing
+  - [ ] Code is peer reviewed and pushed to master
+  - [ ] Deployed successfully to test environment
+  - [ ] Checked against CODING guidelines
```

Comment:
This is too vague to me. Needs some exact specification what they are or remove it

Reply:
I think this refers to the CODING.md in this repo. I add a hyperlink for it.